Directory.Build.props

```diff
@@ -1,31 +1,30 @@
 <Project>
 
   <PropertyGroup>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <WarningsAsErrors />
-    <DebugType>Portable</DebugType>
-    <LangVersion>8.0</LangVersion>
+    <Authors>Microsot Health Team</Authors>
```

Comment:
This value is freeform and doesn't need to be related to any user in the feed/nuget.org. Does anyone have another value they'd like to use here?